src/components/Nav/index.js

```diff
@@ -45,4 +58,14 @@ class Nav extends React.Component {
   }
 }
 
-export default Nav;
+const mapStateToProps = (state) => {}
+
+const mapDispatchToProps = dispatch => ({
+  startSetPledgers: (year) => dispatch(startSetPledgers(year))
```

Comment:
you should pass this function down as a prop instead of connecting it to state here